pkg/lightning/mydump/loader.go

```diff
@@ -823,8 +839,8 @@ func SampleFileCompressRatio(ctx context.Context, fileMeta SourceFileMeta, store
 	return float64(tot) / float64(pos), nil
 }
 
-// SampleParquetDataSize samples the data size of the parquet file.
-func SampleParquetDataSize(ctx context.Context, fileMeta SourceFileMeta, store storage.ExternalStorage) (int64, error) {
+// SampleParquetRowSize samples one row size of the parquet file.
```

Comment:
what do you mean by "one row size"? I think row size is enough

Reply:
should be "row size", thanks!